packages/blocks/src/api/raw-handling/image-corrector.native.js

```diff
@@ -0,0 +1,11 @@
+
+export default function( node ) {
+	if ( node.nodeName !== 'IMG' ) {
+		return;
+	}
+
+	// Remove trackers and hardly visible images.
```

Comment:
Trackers and hardly visible images? What are those? In what case they are present?

Reply:
This is copied from the web code. I image they can have copy paste of HTML that have this things, and they want to filter it out.
Even on mobile it can happen when copying from other apps.

Reply:
Gotcha. Thanks for the explanation